pureStorageDriver/README.md

```diff
@@ -19,7 +19,7 @@ This helm chart installs the Pure Service Orchestrator CSI plugin on a Kubernete
   - Kubernetes 1.13+
     - NOTE: for Kubernetes 1.17, there is an [issue](https://github.com/kubernetes/kubernetes/issues/87852) using vxlan with Flannel or Calico 
   - Minimum Helm version required is 3.1.0.
-  - Google Anthos 1.2.x, 1.3.x (to be verified)
+  - Google Anthos 1.2.x, 1.3.x support the [stateless PSO CSI plugin](https://github.com/purestorage/helm-charts/tree/master/pure-csi) only
```

Comment:
@pure-jliao
We probably don't need to call out "stateless" in our documentation. Stateful and stateless are mostly referenced internally, I think we can just call out `pure-csi` driver 5.x.

Reply:
sorry got merged, probably put it in next change.